diesel_cli/README.md

```diff
@@ -7,17 +7,20 @@ bi-directional changes to your database that get applied sequentially.
 Installation
 ---------------
 
-The diesel cli, by default, requires [`libpq`][postgres], [`sqlite`][sqlite]
+The diesel cli, by default, requires [`openssl`][openssl] [`libpq`][postgres], [`sqlite`][sqlite]
 and [`mysql`][mysql]. Once these dependencies are installed, you can run
 `cargo install diesel_cli`.
 
+> Note: Make sure that both the `bin` and `lib` directories for postgres are added to your PATH
+
 To install the cli without these dependencies, omit the unneeded dependencies from
 the following command:
 
 ```sh
 cargo install diesel_cli --no-default-features --features "postgres sqlite mysql"
 ```
 
+[openssl]: https://indy.fulgan.com/SSL/
```

Comment:
Why isn't it `https://www.openssl.org/source/`?

Reply:
I grabbed it from the Wiki. Also, noticing now that it's Windows binaries only. Will change to that URL